modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -3967,6 +3972,20 @@
 		end
 	end
 
+	function m.linuxMultiProcNumber(cfg)
+		-- Linux equivalent of 'MultiProcessorCompilation'
+		-- Default to 8 parallel jobs
+		if cfg.flags.MultiProcessorCompile then
+			m.element("MultiProcNumber", nil, "8")
+		end
```

Comment:
@nickclark2016 worth keeping this in mind when doing the conversion to API, might be nice to be able to specify a number for the systems that allow it.

Reply:
Yes agreed that's a nice extension to it